[KED-2990] Update ReadMe for Experiment Tracking (#649)
quantumblacklabs/kedro-viz: README.md

## Introduction

> Kedro-Viz is an interactive development tool for building data science pipelines with [Kedro](https://github.com/quantumblacklabs/kedro). Kedro-Viz also allows users to view and compare different experiments from the Kedro project. 

Reviewer: suggestion: `Kedro-Viz also allows users to view and compare different experiments generated from their runs in the Kedro project.`

Reviewer 2: Just discussed with Rashida that `Kedro-Viz also allows users to view and compare different runs in the Kedro project.` would be better.

- 🔬 Focus mode for modular pipeline visualisation
- 📊 Rich metadata side panel to display parameters, plots, etc.
- ♻️ Autoreload on code change

> - 🧪 Supports tracking and comparing experiments from a Kedro projects

Reviewer 2: ```suggestion
- 🧪 Supports tracking and comparing runs from a Kedro project
```

- 🎩 Many more to come

> ## Experiment Tracking on Kedro-Viz 

Reviewer 2: Should it be `on` or `in`? 🤔

Author: in sounds better i think.

> To enable [experiment tracking](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html) on Kedro-Viz, you need to add the Kedro-Viz SQLiteStore to your Kedro project.

Reviewer 3: ```suggestion
To enable [experiment tracking](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html) in Kedro-Viz, you need to add the Kedro-Viz SQLiteStore to your Kedro project.
```

> This can be done by adding the below code to the settings.py in src folder of your project. 

Reviewer 2: ```suggestion
This can be done by adding the below code to `settings.py` in the `src` folder of your Kedro project. 
```

```python
from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
from pathlib import Path
SESSION_STORE_CLASS = SQLiteStore
SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}

> 

Reviewer 3: ```suggestion
```

```

> Once the above set-up is complete, tracking datasets can be used to track different experiments. More information on how to use tracking datasets can be found [here](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html)

Reviewer 2: ```suggestion
Once the above set-up is complete, tracking datasets can be used to track relevant data for Kedro runs. More information on how to use tracking datasets can be found [here](https://kedro.readthedocs.io/en/stable/08_logging/02_experiment_tracking.html)
```

> NOTE - Experiment Tracking is only available for Kedro-Viz versions >= 4.0.2 

Reviewer 2: Maybe also worth mentioning that it's available from Kedro >= 0.17.5
